webpack/components/Search/Search.js

```diff
@@ -48,9 +50,13 @@ const Search = ({
       }
       // Checking whether the current component is mounted before state change events
       if (!mountedRef.current) return;
-      setItems(data?.data?.filter(({ error }) => !error).map(({ label }) => ({
-        text: label.trim(),
-      })));
+      if (endpoint.includes('auto_complete_arch') || endpoint.includes('auto_complete_name')) {
```

Comment:
We should likely set this up as a switch for future use (preference) 

```
switch(true){
case endpoint.includes('auto_complete_arch'): 
case endpoint.includes('auto_complete_name'): 
  setItems(data?.data?.map(label => ({ text: label.trim() })));
default: 
 setItems(data?.data?.filter(({ error }) => !error).map(({ label }) => ({
          text: label.trim(),
        })));
```

Reply:
I'm okay with that. I'll change this.